node/plugin/tests/plugin_test.go

```diff
@@ -20,24 +22,38 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func init() {
-	flag.StringVar(&templateName, "config", "testconfig-anvil.yaml", "Name of the config file (in `inabox/templates`)")
-	flag.StringVar(&testName, "testname", "", "Name of the test (in `inabox/testdata`)")
-}
-
 var (
-	templateName string
-	testName     string
-
 	logger = test.GetLogger()
 
 	// Shared test resources
 	anvilContainer *testbed.AnvilContainer
-	testConfig     *deploy.Config
+	deployResult   *testbed.DeploymentResult
+	privateKeys    *testbed.PrivateKeyMaps
+	testOperator   OperatorConfig
```

Comment:
for a follow-up PR: we should stop using these global vars and instead have the testbed return an actual testbed... which we use in each Test separately.

Reply:
Yeah I want to do this once everything is deployed through testbed (still missing EigenDA components for example).